src/cloud-api-adaptor/test/provisioner/aws/provision_common.go

```diff
@@ -1249,6 +1249,13 @@ func NewEKSCluster(cfg aws.Config, vpc *Vpc, SshKpName string) *EKSCluster {
 func (e *EKSCluster) CreateCluster() error {
 	var err error
 
+	if e.Name != "" {
+		log.Infof("Using existing EKS cluster: %s", e.Name)
+		return nil
+	}
+
+	e.Name = e.Vpc.BaseName + "-k8s"
```

Comment:
Is this always unique, or could the vpc basename be re-used?

Reply:
Hi @stevenhorsman , it can be re-used. You just need to pass the `resources_basename` property.